ui/src/app/components/messages.tsx

```diff
@@ -10,21 +10,21 @@ const MessageWithIcon = ({
   iconPosition,
   children,
 }: {
-  messageType: 'warning' | 'error' | 'success';
+  messageType: 'warning' | 'error' | 'info';
   iconSize: number;
   iconPosition: 'top' | 'center' | 'bottom';
   children;
 }) => {
   const icon = {
     warning: 'warning-standard',
     error: 'warning-standard',
-    success: 'success-standard',
+    info: 'info-standard',
```

Comment:
The `success` option was unused. While I didn't want to add any uses of Clarity icons, swapping "success" for "info" here seemed like a net neutral change.